CWStatusBarNotification/CWStatusBarNotification/CWStatusBarNotification.swift

```diff
@@ -460,13 +460,12 @@ public class CWStatusBarNotification : NSObject {
         
         // checking for screen orientation change
         NSNotificationCenter.defaultCenter().addObserver(self,
-            selector: "updateStatusBarFrame",
+            selector:#selector(CWStatusBarNotification.updateStatusBarFrame),
             name: UIApplicationDidChangeStatusBarFrameNotification,
             object: nil)
         
         // checking for status bar change
-        NSNotificationCenter.defaultCenter().addObserver(self,
-            selector: "updateStatusBarFrame",
+        NSNotificationCenter.defaultCenter().addObserver(self, selector:#selector(CWStatusBarNotification.updateStatusBarFrame),
```

Comment:
Please break up this line like the other notification center bits of code above. Also, please add a space after `selector:`.